packages/core/src/llm/LLM.ts

```diff
@@ -36,7 +37,7 @@ export type MessageType =
   | "memory";
 
 export interface ChatMessage {
-  content: string;
+  content: any;
```

Comment:
This is fine for now, but let's do some typing around this in a different PR.
![Screenshot 2023-11-09 at 3 49 36 PM](https://github.com/run-llama/LlamaIndexTS/assets/1209314/3237f3b9-2708-438b-98d5-25e77550b10d)